pilot/pkg/networking/core/v1alpha3/listener_inbound.go

```diff
@@ -329,6 +361,20 @@ func (lb *ListenerBuilder) buildInboundChainConfigs() []inboundChainConfig {
 			if cc.bind == "" {
 				// If user didn't provide, pick one based on IP
 				cc.bind = getSidecarInboundBindIP(lb.node)
+				// if no global unicast address
+				if cc.bind == WildcardAddress || cc.bind == WildcardIPv6Address {
+					oWildcardAndLocalHost := NewWildcardAndLocalHost(lb.node.GetIPMode())
```

Comment:
This logic seems a bit weird. We call getSidecarInboundBindIP which computes the wildcard address. Then we check if it is a wildcard, we compute the wildcard _again_ (with slightly different logic than getSidecarInboundBindIP/getActualWildcardAndLocalHost) and overwrite it.

Maybe we should just make getSidecarInboundBindIP use NewWildcardAndLocalHost to begin with?

Reply:
I prefer the latter, changed `getSidecarInboundBindIP` into `getSidecarInboundBindIPs`, and use `NewHostAddresses` at the beginning.